frontend/public/features.ts

```diff
@@ -93,10 +94,18 @@ const detectOpenShift = dispatch => coFetchJSON(openshiftPath)
       : handleError(err, FLAGS.OPENSHIFT, dispatch, detectOpenShift)
   );
 
+const projectListPath = `${k8sBasePath}/apis/project.openshift.io/v1/projects`
```

Comment:
Suggest `?limit=1`. You only need to know if there's a single project you can see.